sktime/forecasting/hf_momentfm_forecaster.py

```diff
@@ -0,0 +1,532 @@
+"""Interface for the momentfm deep learning time series forecaster."""
+
+import warnings
+
+import numpy as np
+import pandas as pd
+from skbase.utils.dependencies import _check_soft_dependencies
+
+from sktime.forecasting.base import _BaseGlobalForecaster
+
+if _check_soft_dependencies(["momentfm", "torch"], severity="none"):
+    import momentfm
```

Comment:
I think, we need to discuss on how to install momentfm. Does momentfm have a pypi package or is this also dependent on #6779 ?

Reply:
It seems that there is an official pypi repo, so not affected by #6779

see https://github.com/moment-timeseries-foundation-model/moment#-usage

Reply:
Yup, ive installed the package locally for testing and everything looks good so far